Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM circleci/python:3.7.0-stretch-browsers
```

Comment:
[This image](https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/python/images/3.7.0-stretch/browsers/Dockerfile) installs firefox and chrome that we don't need since we install them manually in addition. Maybe using `circleci/python:3.7.0-strech` could be enough (and possibly [setting up `DISPLAY` and `Xvfb`]( https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/python/images/3.7.0-stretch/browsers/Dockerfile#L64))

Reply:
Thanks.  I couldn't find the recipe for `DISPLAY` and `Xvfb`, which lead me to the `browsers` image.